kolibri/plugins/learn/package.json

```diff
@@ -3,6 +3,10 @@
   "description": "Learn Plugin for Kolibri",
   "private": true,
   "version": "0.0.1",
+  "dependencies": {
+    "deep-object-diff": "^1.1.9",
```

Comment:
Not related to your work here, but I'm curious if `lodash` can accomplish what this dependency does